samples/GeneratorInConsumerSolution/nuget.config

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
+    <add key="local CG.R Debug packages" value="../../bin/Packages/Debug" />
```

Comment:
Using Debug packages unconditionally is unfortunate. This would fail if someone (including the build CI if it starts validating this) built Release, or (worse) cause unexpected results if they built Debug a long time ago, made changes and cleared the nuget cache, and built in release and still saw the old debug behavior.
I believe nuget sources can be added as msbuild items, and while I generally hate that pattern, in this case it might make sense so that you can add the path with `$(Configuration)` used.

Reply:
Good suggestion! Found it: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#restore-properties

Reply:
I've tried but found a problem: https://github.com/AArnott/CodeGeneration.Roslyn/pull/152/commits/a948f29212c11f2f4c5c482e647368e1eefbf49b

DotNetCliToolReference isn't working with that scenario, at least I suspect that's the cause.

Reply:
OK, well thanks for trying. What you have here will work most of the time. :)